playwright/plugins/docker/index.ts

```diff
@@ -18,6 +18,35 @@ import * as os from "os";
 import * as crypto from "crypto";
 import * as childProcess from "child_process";
 import * as fse from "fs-extra";
+import { ExecFileException } from "child_process";
+
+const exec = (
+    cmd: string,
+    args: string[],
+    pipe = true,
+    callback?: (error: ExecFileException | null, stdout: string, stderr: string) => void,
+): Promise<{ stdout: string; stderr: string }> => {
```

Comment:
I can't really figure out what `callback` and `pipe` do here, or how `callback` relates to the result. Document, please.